lib/Myriad/Transport/Redis.pm

```diff
@@ -303,7 +303,7 @@ method pending(%args) {
     Future->wait_any(
         $src->completed->without_cancel,
         (async sub {
-            $instance = await $self->redis_from_pool;
+            $instance = await $self->borrow_instance_from_pool;
```

Comment:
general naming convention: if something returns a type, that type should be on the left.

In this case, the important part is `redis`, that's why it's on the left.

Reply:
is not having `$redis->redis_from_pool` (https://github.com/binary-com/perl-Myriad/pull/46/files#diff-4cc9d61193d402e5c6d6a55390b39413a32c331e022eee9b857fc4e835d54873R88) 
a good point to start with?

Reply:
is not having $redis->redis_from_pool (https://github.com/binary-com/perl-Myriad/pull/46/files#diff-4cc9d61193d402e5c6d6a55390b39413a32c331e022eee9b857fc4e835d54873R88)
a good point to start with?